src/app/components/Application/Application.jsx

```diff
@@ -1,7 +1,13 @@
+import { hot } from 'react-hot-loader';
 import React from 'react';
 
-import { Header, navConfig } from '@nypl/dgx-header-component';
-import Footer from '@nypl/dgx-react-footer';
+/**
+ * TODO:
+ * Currently, the NYPL Header and Footer are not using React 16
```

Comment:
Probably should say in README that if a team wants to use this boilerplate that the new app can't have Header and Footer.